src/components/molecules/FormEditor/FormEditor.tsx

```diff
@@ -140,6 +144,7 @@ const FormEditor = (props: {formSchema: any; formUiSchema?: any}) => {
           const filePath = getAbsoluteFilePath(selectedPath, fileMap);
           const fileContent = fs.readFileSync(filePath, 'utf8');
           const content = mergeManifests(fileContent, formString);
+          setIsResourceUpdated(content.trim() !== fileContent.trim());
```

Comment:
perhaps move this to after the writeFileSync to avoid having to do the comparison twice? - same goes above